test/jasmine/tests/axes_test.js

```diff
@@ -2851,6 +2851,57 @@ describe('Test axes', function() {
             ]);
         });
 
+        it('Does not use SI prefixes for 10^n with |n| < minexponent', function() {
+            var textOut = mockCalc({
+                type: 'log',
+                tickmode: 'linear',
+                exponentformat: 'SI',
+                minexponent: 5,
+                showexponent: 'all',
+                tick0: 0,
+                dtick: 1,
+                range: [-18.5, 18.5]
+            });
+
+            expect(textOut).toEqual([
+                '10<sup>\u221218</sup>',
+                '10<sup>\u221217</sup>',
+                '10<sup>\u221216</sup>',
+                '1f', '10f', '100f', '1p', '10p', '100p', '1n', '10n', '100n',
+                '1μ', '0.00001', '0.0001', '0.001', '0.01', '0.1', '1', '10', '100',
+                '1000', '10,000', '100,000', '1M', '10M', '100M', '1G', '10G', '100G',
+                '1T', '10T', '100T',
+                '10<sup>15</sup>',
+                '10<sup>16</sup>',
+                '10<sup>17</sup>',
+                '10<sup>18</sup>'
+            ]);
+            var textOut = mockCalc({
```

Comment:
`textOut` is already defined here.
That's the reason why the syntax test is failing. So let's drop extra `var` here.
To ensure the syntax test is passing, you may consider running the syntax test locally using the following command:
```
npm run test-syntax
```

Reply:
Thanks, fixed in f74e17f7039e1dae8a5450e88074f3123ae85b01 .

I'll try to run the tests but I'm having trouble due to my old version of node (which I can't upgrade because my company uses a RedHat release with old glibc).